dds/idl/value_helper.h

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
This file should probably be in `dds/DCPS` because there are several headers in `dds/DCPS` that we include in `dds/idl`, but I don't think there's any that do the opposite of that.